File: competitive_programming/codeforces/06_javascript/231a_team.js
function team(problems) {
  return problems.filter(([a, b, c]) => a + b + c >= 2).length;
}

if (require.main === module) {
  if (team([[1, 1, 0], [1, 1, 1], [1, 0, 0]]) !== 2) throw new Error("team failed");
  console.log("231A team ok");
}

File: competitive_programming/codeforces/09_rust/231a_team.rs
fn team(problems: &[[i32; 3]]) -> usize {
    problems.iter().filter(|p| p[0] + p[1] + p[2] >= 2).count()
}

fn main() {
    assert_eq!(team(&[[1, 1, 0], [1, 1, 1], [1, 0, 0]]), 2);
    println!("231A team ok");
}
